x/merkledb/db.go

```diff
@@ -1215,5 +1215,5 @@ func cacheEntrySize(p path, n *node) int {
 	if n == nil {
 		return len(p)
 	}
-	return len(p) + len(n.bytes())
+	return len(p) + 2*len(n.bytes())
```

Comment:
Can we put a comment here saying that this is intended to be an approximation of this node's memory footprint and not the exact size?